apps/playground/src/stories/hooks/useInterval.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
```

Comment:
```suggestion
import React, { useState } from 'react';
```

this will fix the linter issue on CI validations 👌🏻 

![image](https://user-images.githubusercontent.com/17727170/236046886-e2c5586a-492d-45c9-91b4-a04c823219f4.png)